src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java

```diff
@@ -63,17 +63,37 @@ private void process() {
 				empRowCount++;
 			}
 			break;
-		case 3:
-			System.out.println("How many records do you want to insert ? : ");
-			int custCount = scanner.nextInt();
+		case 2:
+			System.out.println("###### " + DriverTypes.CUSTOMER.getName().toUpperCase() + " Processor Starts ######");
+
+			
+			Arrays.stream(OperationTypes.values()).forEach(eachOperation -> {
+				System.out.println(eachOperation.getOperationType() + " : " + eachOperation.getOperationName());
+			});
+			System.out.println("Please choose the operation type: ");
+			int operationTypeCustomer = scanner.nextInt();
+
+			if (operationTypeCustomer == 1) {
+
+				System.out.println("How Many Records do you want to insert ? :");
+				int customerCount = scanner.nextInt();
 
 			CustomerDataProcessor custProcessor = new CustomerDataProcessor(scanner, custDao);
 			int custRowCount = 1;
-			for (int i = 0; i < custCount; i++) {
+			for (int i = 0; i < customerCount; i++) {
 				custProcessor.process(custRowCount);
 				custRowCount++;
 			}
-		case 4:
+			System.out.println(customerCount + " records inserted successfully.");
+			} else if (operationTypeCustomer == 2) {
+			  System.out.println("fetching customer records from the database...");
+			  CustomerInsertDao custInsertDao = new CustomerInsertDao();
```

Comment:
why are you are not using custDao which is declared and initialized in instance block  ???? instead creating an object???